### PR TITLE
feat(tutorial): complete step 1

### PR DESCRIPTION
**Pull request by Disput**
Closes #

{{short description}}

#### Changelog

**New**

- {{new thing}}

**Changed**

- {{change thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

File: src/index.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+@import 'carbon-components/scss/globals/scss/styles.scss';
+
+$feature-flags: (
```
Comment (kellychurchill) [inline src/index.scss]: This mapping should come before the styles import. 
https://www.carbondesignsystem.com/tutorial/react-step-1#add-ui-shell-sass

File: src/app.scss [hunk 2]
```diff
@@ -0,0 +1,4 @@
+@import './components/TutorialHeader/tutorial-header.scss';
+@import './components/TutorialHeader/tutorial-header.scss';
```
Comment (kellychurchill) [inline src/app.scss]: You have a duplicate import here.

**Comment by Disput** [comment]
sorry, fixed

File: src/app.scss (hunk 2 above)
Comment (kellychurchill) [reply src/app.scss]: This is still not resolved and your Netlify preview build is failing: https://app.netlify.com/sites/carbon-tutorial/deploys/5d4c044d57bd6b0007a9cbb7

Comment (Disput) [reply src/app.scss]: My view says something different
[https://github.com/carbon-design-system/carbon-tutorial/pull/2350/files#diff-21e3718002c800ff2f9682b9f454a67f]

Comment (Disput) [reply src/app.scss]: the preview build is from 8 August, do you have a current one?

Comment (kellychurchill) [reply src/app.scss]: https://app.netlify.com/sites/carbon-tutorial/deploys/5d513797c1a9fe000862c5fc

**Comment by Disput** [comment]
I repeated all the steps from blank branch. Still get the ci-check error `Invariant failed: You should not use <Link> outside a <Router>`

File: package.json [hunk 3]
```diff
@@ -7,7 +7,7 @@
   "license": "Apache-2.0",
   "scripts": {
     "build": "react-scripts build",
-    "ci-check": "yarn format:diff",
+    "ci-check": "yarn format:diff && yarn test",
```
Comment (kellychurchill) [inline package.json]: You should not have `yarn test` here

File: src/App.css [hunk 4]
```diff
@@ -0,0 +1,33 @@
+.App {
```
Comment (kellychurchill) [inline src/App.css]: There should not be a `src/App.css` file committed

File: src/App.js [hunk 5]
```diff
@@ -1,12 +1,24 @@
 import React, { Component } from 'react';
+import './app.scss';
+import { Button } from 'carbon-components-react';
```
Comment (kellychurchill) [inline src/App.js]: This import is not needed.

File: src/components/TutorialHeader.js [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+const TutorialHeader = () => (
+  <Header aria-label="Carbon Tutorial">
+    <SkipToContent />
+    <HeaderName element={Link} to="/" prefix="IBM">
+      Carbon Tutorial
+    </HeaderName>
+    <HeaderNavigation aria-label="Carbon Tutorial">
+      <HeaderMenuItem element={Link} to="/repos">
+        Repositories F
```
Comment (kellychurchill) [inline src/components/TutorialHeader.js]: Errant "F"

**Comment by Disput** [comment]
I really thought I did this in the previous step .... sorry for the additional work

**Pull request closed without merging**